TABTestKit/Classes/Elements/DatePicker.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  DatePicker.swift
+//  Pods-TABTestKit_Example
+//
+//  Created by Annie Fung on 06/12/2019.
+//
+
+import XCTest
+
+/// Represents a datepicker.
+/// To interact with the wheels you must first
+/// ask the picker for one of it's wheels, by index.
+
+public struct DatePicker: Element {
```

Comment:
Docs won't show properly in Xcode's help window if there's whitespace between the documentation and the struct, so can we remove line 13?

Reply:
done